components/modal/borrow-more-modal.tsx

```diff
@@ -34,59 +41,111 @@ const BorrowMoreModal = ({
   position,
   onClose,
 }: {
-  position: {
-    currency: Currency
-    amount: string
-  } | null
+  position: LoanPosition
```

Comment:
`LoanPosition | null`

Reply:
<img width="921" alt="image" src="https://github.com/clober-dex/coupon.finance/assets/10525011/6f180813-0117-43e8-abcb-97cacb55d708">

I've already implemented a null check here: [Link](https://github.com/clober-dex/coupon.finance/blob/feat/borrow-more-view/components/borrow.tsx#L424). It seems like fixing all occurrences of position potentially being undefined is a critical safety measure.

Reply:
I missed that. Looks good! Shouldn't we use `position?` -> `position` then?